src/style.css

```diff
@@ -0,0 +1,96 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: AvenirBold;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+.search-form {
+  margin: 20px 0;
+  position: relative;
+}
+
+.input {
+  box-sizing: border-box;
+  height: 70px;
+  width: 100%;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  padding-left: 62px;
+  line-height: 22px;
+  font-size: 16px;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.icon {
+  position: absolute;
+  width: 19px;
+  bottom: 26px;
+  left: 26px;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  outline: none;
+  font-family: "AvenirBold", sans-serif;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.input::placeholder {
+  font-family: "Avenir", sans-serif;
+}
+
+.search-form-small {
+  position: relative;
+}
+
+.input-small {
+  height: 42px;
+  box-sizing: border-box;
+  width: 100%;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  padding-left: 33px;
+  line-height: 19px;
+  font-size: 14px;
+  color: #3d4e61;
+  background: #fff;
```

Comment:
background is a shorthand property, use background-color
https://developer.mozilla.org/en-US/docs/Web/CSS/background